front-end/src/components/main-page/OverrideModal.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Modal from 'react-modal';
+import BaseButton from '../general/BaseButton';
+import PropTypes from 'prop-types';
+
+class OverrideModal extends React.Component {
+
+    static propTypes = {
+        show: PropTypes.bool.isRequired,
+        toggleModal: PropTypes.func.isRequired,
+        uploadFileWithOverride: PropTypes.func.isRequired,
+    }
+
+    async overrideFile(){
+        this.props.uploadFileWithOverride();
+        this.props.toggleModal();
+    }
+
+    componentDidMount() {
+        Modal.setAppElement("body");
+    }
+
+    render() {
+        return (
+            <Modal
+                isOpen={this.props.show}
+                className="container w-40 md:w-60 mx-auto bg-purple-100 rounded-lg shadow-lg py-3 mt-[40vh] flex flex-col"
```

Comment:
```suggestion
                className="container w-40 md:w-60 mx-auto bg-purple-100 font-cabin rounded-lg shadow-lg py-3 mt-[40vh] flex flex-col"
```
Seeing this made me realize that while styling I forgot to also change the font in the modals lol, that's myb. If you can, could you also change the font in the upload modal? tyyy.